src/windows/gfx/d3d11.c

```diff
@@ -301,6 +302,134 @@ static bool d3d11_refresh_resource(struct gfx *gfx, MTY_Device *_device, MTY_Con
 	return r;
 }
 
+static bool d3d11_map_shared_resource(struct gfx *gfx, MTY_Device *_device, MTY_Context *context, MTY_ColorFormat fmt,
+	uint8_t plane, const uint8_t *image, uint32_t full_w, uint32_t w, uint32_t h, uint8_t bpp)
+{
+
+	// We do all textures at once so we dont want to deal with any later planes.
+	if (plane > 0)
+		return true;
+
+	struct d3d11 *ctx = (struct d3d11 *) gfx;
+	bool r = true;
```

Comment:
This one can be removed, since after the for loop, it will return true, and all except cases will return false.